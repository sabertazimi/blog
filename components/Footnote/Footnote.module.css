.footnote {
  @apply mt-72 mb-9 !important;
}

.footnote h2,
.footnote div,
.footnote p,
.footnote li {
  @apply my-0 !important;
  @apply text-gray-500 text-sm !important;
}
