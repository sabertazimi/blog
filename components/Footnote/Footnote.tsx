import type { ReactNode } from "react";
import {classNames}from '@components/utils';
import styles from './Footnote.module.css';

interface Props {
  children?: ReactNode;
  className?: string;
}

const Footnote = ({children, className, ...props}:Props):JSX.Element => (
  <div {...props} className={classNames(className, styles.footnote)}>
    {children}
  </div>
)

export default Footnote;
