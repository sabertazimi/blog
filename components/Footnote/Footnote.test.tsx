import { render } from '@testing-library/react';
import Footnote from './Footnote';

describe('Footnote', () => {
  test('should render correctly (snapshot)', () => {
    const { container } = render(<Footnote />);

    expect(container).toMatchSnapshot();
  });
});
