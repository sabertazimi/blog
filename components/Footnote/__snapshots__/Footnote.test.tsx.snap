// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footnote should render correctly (snapshot) 1`] = `
<div>
  <div
    class="footnote"
  />
</div>
`;
